src/devices/machine/gt64xxx.cpp

```diff
@@ -16,26 +17,28 @@
 #define LOGTIMERS(...)      LOGMASKED(LOG_TIMERS, __VA_ARGS__)
 #define LOGDMA(...)         LOGMASKED(LOG_DMA, __VA_ARGS__)
 #define LOGPCI(...)         LOGMASKED(LOG_PCI, __VA_ARGS__)
+#define LOGIRQ(...)         LOGMASKED(LOG_IRQ, __VA_ARGS__)
 
 
-/*************************************
- *
- *  Galileo constants
- *
- *************************************/
+//************************************
+//
+//  Galileo constants
+//
+//************************************
 
 #define TIMER_PERIOD        attotime::from_hz(clock())
 #define PCI_BUS_CLOCK       33000000
-// Number of dma words to transfer at a time, real hardware configurable between 8-32
+// Number of dma dwords to transfer at a time, real hardware configurable between 8-32
```

Comment:
This is a MIPS host - words are 32 bits.  Is this a number of 64-bit double-words, or 32-bit words?  I don’t want Intel terminology being forced into places it doesn’t belong.